### ISSUE TITLE
Formatting Ribbon below the title bar of active note

### ISSUE THREAD

**Issue by viaahmed** [Reocin/obsidian-markdown-formatting-assistant-plugin#2]
**Is your feature request related to a problem? Please describe.**
It would be easier it was in the note instead of a different panel.

**Describe the solution you'd like**
Like in all other apps, can you allow user to set the ribbon just below the note title, or vertically beside the gutter.

Thanks for making this plugin.
Psst, I have not installed it yet, I am away from obsidian now, I made this issue as soon l discovered it.

**Comment by Reocin** [comment]
Hello viaahmed

Thanks for your ideas, but I need more information. I'm really new to Obsidian and uses it just since a couple of days.

> Like in all other apps, can you allow user to set the ribbon just below the note title, or vertically beside the gutter.
Where is this position

Do you mean the position like in the following picture ? Actually I wanted to add the panel with the buttons there, but I don't know how. Do you have some information to help me with that ? 

But anyway mostly you will use the command language ;-) It's much faster to work with.

![Screenshot from 2021-01-30 14-29-07](https://user-images.githubusercontent.com/35258423/106357680-db4b5400-6307-11eb-935c-e79625c5badc.png)

**Comment by viaahmed** [comment]
Yes, I meant exactly as in the picture, I am no programmer, I know
little CSS, I suggest, you can create(render) a ::after pseudo element
below the title bar of the active note. And put all those icons in
that.

**Comment by Reocin** [comment]
Ok i will try, but i can't promise :-)
